### ISSUE TITLE
Minification breaks 'devices' card

### ISSUE THREAD

**Issue by browntownington** [seud0nym/tch-gui-unhide#206]
Hi there!

Outstanding job on these mods! I just grabbed a couple cobraxh's and got to know the mods pretty well after many rebuilds.

Only found two issues.

Local devices card won't open after minification is done. The spin wheel loads quick then the screen overlays transparent screen but nothing happens. A click on the screen returns you back to card view. 

Tried several browsers / modems etc. bug occurs each time.

**Comment by seud0nym** [comment]
Have you tried the latest test release? You can apply it with:
```
./tch-gui-unhide -u && ./tch-gui-unhide -y
```

There are a lot of chnages to the Devices screen, so I think it might already be fixed...

Please let me know how you go.

**Comment by browntownington** [comment]
Thanks mate! That did the trick :D

You whats funny. I saw another issue similarly, however you mentioned it was fixed in the latest version. Since I believed I was on the latest version after running the install curl I assumed it must be another issue.

So the install curl as per your install instructions 'curl -skL https://raw.githubusercontent.com/seud0nym/tch-gui-unhide/master/get | sh -s --' gets the same version that's listed as your current release 31.05.2024.

But after running './tch-gui-unhide -u' it's pulling a newer version 04.07.2024 magically out of thin air 🤣
